packages/Modal/src/Modal.tsx

```diff
@@ -192,15 +192,14 @@ const Modal: React.FunctionComponent<ModalProps> = (props: ModalProps) => {
                                 >
                                     {displayBackBtn ? (
                                         <IconButton
-                                            size="small"
                                             className="ids-modal__back"
                                             onClick={() => {
                                                 if (carousel && carousel.currentSlide) {
                                                     handleOnPageChange(carousel.currentSlide - 1);
                                                 }
                                             }}
                                             appearance={{ type: "ghost", variant: "secondary" }}
-                                            icon={<ChevronLeft size="medium" />}
+                                            icon={<AngleLeftIcon />}
```

Comment:
they are basically the same thing. I figured it doesn't really matter.. They should be the same size. We have no way of knowing which is Igloo and which is workleap theme inside the component unless we pass a prop, I figured it would be way more work for nothing. No one would really notice.